sigh/lib/sigh/download_all.rb

```diff
@@ -23,7 +23,7 @@ def download_profile(profile)
       type_name = profile.class.pretty_type
       type_name = "AdHoc" if profile.is_adhoc?
 
-      profile_name = "#{type_name}_#{profile.app.bundle_id}.mobileprovision" # default name
+      profile_name = "#{type_name}_#{profile.uuid}.mobileprovision" # default name
```

Comment:
can you please add the   `profile.app.bundle_id` too -  so that we see the bundle id in the filename too.